bedrock/mozorg/templates/mozorg/home/includes/m24/products.html

```diff
@@ -11,37 +11,37 @@ <h2 class="m24-c-copy-title">Surf the web, on your terms</h2>
   </div>
   <ul class="m24-c-spring">
     <li class="m24-c-spring-item">
-      <a class="m24-c-spring-link" href="{{ url('firefox') }}" data-cta-text="Firefox" data-cta-position="product-list">
+      <a class="m24-c-spring-link m24-t-product-firefox" href="{{ url('firefox') }}" data-cta-text="Firefox" data-cta-position="product-list">
         <strong class="m24-c-spring-title">Firefox</strong>
         <span class="m24-c-spring-info">Browse with speed, privacy, and control.</span>
       </a>
     </li>
     <li class="m24-c-spring-item">
-      <a class="m24-c-spring-link" href="https://www.fakespot.com/{{ utm_params }}" data-cta-text="Fakespot" data-cta-position="product-list">
+      <a class="m24-c-spring-link m24-t-product-fakespot" href="https://www.fakespot.com/{{ utm_params }}" data-cta-text="Fakespot" data-cta-position="product-list">
         <strong class="m24-c-spring-title">Fakespot</strong>
         <span class="m24-c-spring-info">Spot fake reviews, products, and maybe one day fake people. </span>
       </a>
     </li>
     <li class="m24-c-spring-item">
-      <a class="m24-c-spring-link" href="https://getpocket.com/signup{{ utm_params }}" data-cta-text="Pocket" data-cta-position="product-list">
+      <a class="m24-c-spring-link m24-t-product-pocket" href="https://getpocket.com/signup{{ utm_params }}" data-cta-text="Pocket" data-cta-position="product-list">
         <strong class="m24-c-spring-title">Pocket</strong>
         <span class="m24-c-spring-info">Save content and read it later on any device, online or off.</span>
       </a>
     </li>
     <li class="m24-c-spring-item">
-      <a class="m24-c-spring-link" href="{{ url('products.vpn.landing') }}" data-cta-text="VPN" data-cta-position="product-list">
+      <a class="m24-c-spring-link m24-t-product-vpn" href="{{ url('products.vpn.landing') }}" data-cta-text="VPN" data-cta-position="product-list">
         <strong class="m24-c-spring-title">VPN</strong>
         <span class="m24-c-spring-info">Stay safe on public Wi-Fi, block advertisers, and more.</span>
       </a>
     </li>
     <li class="m24-c-spring-item">
-      <a class="m24-c-spring-link" href="https://monitor.mozilla.org/{{ utm_params }}" data-cta-text="Monitor" data-cta-position="product-list">
+      <a class="m24-c-spring-link m24-t-product-monitor" href="https://monitor.mozilla.org/{{ utm_params }}" data-cta-text="Monitor" data-cta-position="product-list">
         <strong class="m24-c-spring-title">Monitor</strong>
         <span class="m24-c-spring-info">Check if your data has been breached then make it private.</span>
       </a>
     </li>
     <li class="m24-c-spring-item">
-      <a class="m24-c-spring-link" href="https://www.thunderbird.net/{{ utm_params }}" data-cta-text="Thunderbird" data-cta-position="product-list">
+      <a class="m24-c-spring-link m24-t-product-thunderbird" href="https://www.thunderbird.net/{{ utm_params }}" data-cta-text="Thunderbird" data-cta-position="product-list">
```

Comment:
**Nit:** I also noticed the design uses a different Thunderbird image?

![Screenshot 2024-09-13 at 12 19 50 PM](https://github.com/user-attachments/assets/60e4963e-e4d9-46d8-b0d3-fc20806b8c49)

Reply:
This one seems wrong and broken to me?

Reply:
Yeah it looked like a variation I haven't seen before

Reply:
The current TB logo design SVG uses gradients and filters that a) often break on edits, leaving weird artifacts, b) trigger known bugs in some image libraries, messing up the colors… 

So seeing this "blackbird" has some kind of an unusual element appear down over the envelope, this is most likely a broken rendering of the logo SVG, after cropping or resizing it and working with it within a bigger document.